src/libraries/System.Data.Common/src/System/Xml/XmlDataDocument.cs

```diff
@@ -1102,22 +1075,39 @@ private XmlNode CloneNode(DataPointer dp) =>
                 _ => throw new InvalidOperationException(SR.Format(SR.DataDom_CloneNode, dp.NodeType.ToString())),
             };
 
-        internal static bool IsTextLikeNode(XmlNode n)
+        internal static class Helpers
```

Comment:
I'd add a comment why this is a separate class... to avoid future discussions and somebody trying to refactor it back :-)